tests/foreman/cli/test_repository.py

```diff
@@ -3270,3 +3270,38 @@ def test_copy_erratum_and_RPMs_within_a_date_range():
 
     :CaseImportance: Medium
     """
+
+
+@pytest.mark.tier2
+def test_positive_syncable_yum_format_repo_import(target_sat, module_org):
+    """Verify that you can import syncable yum format repositories
+
+    :id: dc6422c3-1ea1-4856-ab64-7c66e6043300
+
+    :steps:
+        1. Execute hammer command
+            hammer content-import repository --organization='' --path=''
+        2. Assert job is successful
+        3. Verify repository is synced and contains rpms
+
+    :expectedresults: The syncable repository sync correctly and all content is imported
+
+    :customerscenario: true
+
+    :BZ: 2128894
+    """
+    target_sat.execute('mkdir /var/lib/pulp/imports/syncable-repo')
+    target_sat.execute(
+        'wget -r -np -nH -e robots=off -P /var/lib/pulp/imports/syncable-repo'
+        f' {settings.robottelo.REPOS_HOSTING_URL}/exported_repo/'
+    )
+    synced_repo = target_sat.execute(
+        f'hammer content-import repository --organization={module_org.name}'
+        ' --path=/var/lib/pulp/imports/syncable-repo/exported_repo'
+    )
```

Comment:
We have https://github.com/SatelliteQE/robottelo/blob/master/robottelo/cli/content_import.py that you can access through `target_sat.cli`. All you need to do is add the `repository` method to give us access to that subcommand like this:
```python
@classmethod
def repository(cls, options, timeout=None):
    """
    Make a repository import
    """
    cls.command_sub = 'repository'
    return cls.execute(cls._construct_command(options), output_format='json', timeout=timeout)
```
and then in the test you could do something like:
```python
target_sat.cli.ContentImport.repository(
    {'organization': module_org.name, 'path': '/var/lib/pulp/imports/syncable-repo/exported_repo'}
)
```

It's always better to run hammer through `target_sat.cli`